astropy/io/fits/convenience.py

```diff
@@ -463,6 +462,8 @@ def table_to_hdu(table):
 
     # Not all tables with mixin columns are supported
     if table.has_mixin_columns:
+        from ...table.column import BaseColumn
```

Comment:
More style nitpicks:
- Comment is *always*: `#` followed by <space> followed by comment that starts with a capital letter.
- Keep within 80 characters (ok to break line)
- Usually don't start with an article like "A" or "The".
- If the comment is a full sentence then end with a period.

So:
```
        # Import is done here, in order to avoid it at build time as erfa is not 
        # yet available then.
```
Sorry for being picky, but with hundreds of contributors we need to be strict to maintain the standards.  It will become second nature soon!  :smile: